core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/config/AbstractFlowConfiguration.java

```diff
@@ -132,7 +132,11 @@ private AbstractAction getAction(Class<? extends AbstractAction> clazz) {
     }
 
     private AbstractAction getAction(String name) {
-        return applicationContext.getBean(name, AbstractAction.class);
+        try {
+            return applicationContext.getBean(name, AbstractAction.class);
+        } catch (NoSuchBeanDefinitionException ex) {
+            return null;
```

Comment:
I guess if we have state without action we postpone the problems from initial time to runtime. If i'm true, i'm not happy with that.